src/inbox/mod_inbox.erl

```diff
@@ -65,6 +65,11 @@ start(Host, Opts) ->
     {ok, _} = gen_mod:start_backend_module(?MODULE, Opts, callback_funs()),
     mod_disco:register_feature(Host, ?NS_ESL_INBOX),
     IQDisc = gen_mod:get_opt(iqdisc, Opts, no_queue),
+    MucTypes = gen_mod:get_opt(groupchat, Opts),
+    case MucTypes of
```

Comment:
To be future proof:

```erlang
MucTypes = gen_mod:get_opt(groupchat, Opts, []),
lists:member(muc, MucTypes)
```

Reply:
and check that the default option is consistent with `muclight_enabled/1` function below